Project.toml

```diff
@@ -11,17 +11,19 @@ Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
 HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
 InfrastructureSystems = "2cd47ed4-ca9b-11e9-27f2-ab636a7671f1"
+IterTools = "c8e1da08-722c-5040-9ed9-7db0dc04731e"
```

Comment:
What functions are you using from this package? I probably missed it; just saw stdlib functions.

Reply:
I am using product and repeat to generate the permutations.

Reply:
Those are in the standard library.
```
julia> Base.Iterators.product, Base.Iterators.repeated
(Base.Iterators.product, Base.Iterators.repeated)
```